tests/system/Helpers/FormHelperTest.php

```diff
@@ -261,10 +261,33 @@ public function testFormInput()
         $this->assertSame($expected, form_input($data));
     }
 
+    public function testFormInputXHTML()
+    {
+        $doctypes        = config('DocTypes');
+        $default         = $doctypes->html5;
+        $doctypes->html5 = false;
+
+        $expected = <<<EOH
+            <input type="text" name="username" value="johndoe" id="username" maxlength="100" size="50" style="width:50%"/>\n
```

Comment:
The current helpers output one space. So please fix to output the same.

```suggestion
            <input type="text" name="username" value="johndoe" id="username" maxlength="100" size="50" style="width:50%" />\n
```